Theme.h

```diff
@@ -0,0 +1,470 @@
+#pragma once
+#include <vector>
+#include "imgui.h"
+#include "imgui_internal.h"
+#include "mq/base/String.h"
+
+constexpr int ThemeCount = 7;
+
+#pragma region Themes Set Functions
+
+// Default Theme Calls ImGui::StyleColorsDark() to reset the theme back to the MQ default.
+inline void StyleColorsDefault()
+{
+	ImGui::StyleColorsDark();
+}
+
+inline void StyleColorsGrape()
+	{
+	ImGuiStyle*                            style  = &ImGui::GetStyle();
+	ImVec4*                                colors = style-> Colors;
```

Comment:
This is replacing the existing style. You don't want to do that. You want to create your own ImGuiStyle.

Applying your style simply becomes replacing the current style with your own while you are inside your window code, and then restoring the old style.

something like...

```
ImGuiStyle oldStyle = ImGui::GetStyle();
ImGuui::GetStyle() = StyleColorsGrape();

... window code

ImGui::GetSTyle() = oldStyle
```

I think that shuld work

Reply:
ahh ok will try that when i get home. 
I found the ones from style editor drop down and thought that was the way to do it.